src/Widgets/PersonCardWidget/PersonCardWidgetComponent.tsx

```diff
@@ -34,7 +34,7 @@ provideComponent(PersonCardWidget, ({ widget }) => {
         tag="h6"
         className="h6 text-uppercase"
       />
-      <div className="card mb-2 bg-white max-width-350">
+      <div className="card mb-2 bg-white no-color-adaption max-width-350">
```

Comment:
If I visit https://edit.scrivito.com/d0a154d76edf2a7bd991fc658e700a1d~https://scrivito-portal-app.pages.dev/colors?_scrivito_workspace_id=q6a733838df01858&_scrivito_display_mode=view I can't see a different to your proposed changes. Why is `no-color-adapation` needed for the PersonCard widget?

Reply:
When you visit the preview deployment you see the issue when not have this additional class.
I fixed some heading styles in cards as well.

https://edit.scrivito.com/d0a154d76edf2a7bd991fc658e700a1d~https://agessler-color-sizes-fixes

![Bildschirmfoto 2024-02-09 um 14 39 57](https://github.com/Scrivito/scrivito-portal-app/assets/710311/1ec94dde-1676-43ac-8555-a29a667476a8)
scrivito-portal-app.pages.dev/colors

Reply:
I need this extra class because in some cases you place a `PersonCard` or `DownloadCard` within another card. The only chance to solve this easily is to write it twice or use `>` in CSS. Second can't be used because we do have to many `<div>` bitween these cards.